tests/kuttl/tests/ovn_restart_flow/03-assert.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Check for:
+#
+# - The current bridges flows are present in the logs; these flows writen in
+#   the logs are the ones restored from the previous "ovs-vswitchd" container
+#   execution.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+commands:
+    - script: |
+        controller_pod=$(oc get pod -n $NAMESPACE -l service=ovn-controller-ovs -o name|head -1)
```

Comment:
me not sure if we will always get the re created pod here, if its not guaranteed we can get based on the node name, get sorted node names and get ovs pod on the first node. If it always returns re created pod then we are fine here.

Reply:
I don't think there's a guarantee for the order of the list. Good catch. If it's multinode environment, then we need to stick to a single node. It doesn't matter which it is, so just pick one pod from the list, remember its node name (not sure how to pass data across steps though), then wait for the new pod on the same node and collect its logs.

Reply:
To be honest, right now I have no idea of how to implement that. I don't know how to pass a variable between kuttl test steps.

Reply:
We can instead do use below at both step 2 and 3:-
NODE=$(oc get nodes -o name|sort|head -1| sed "s|node/||g")
controller_pod=$(oc get pod -l service=ovn-controller-ovs -o name --field-selector spec.nodeName=$NODE)

Reply:
Right! That will give us always the first node of the list. Thanks!